src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,8 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Successfully fetched {} products from shopping cart for userId {}",
+                products.size(), userId);
```

Comment:
The log message in this line is informative, which is good. However, it would be better to include more details about the products fetched, such as their names and prices, to make the log more useful.